src/helpers/getByAPI.js

```diff
@@ -112,20 +117,29 @@ export const getByPlaceholderText = (instance: ReactTestInstance) =>
       );
     } catch (error) {
       throw new ErrorWithStack(
-        prepareErrorMessage(error),
+        `${prepareErrorMessage(error)} with placeholder text '${
+          typeof placeholder === 'string' ? placeholder : placeholder.toString()
+        }'`,
```

Comment:
```suggestion
        `${prepareErrorMessage(error)} with placeholder '${placeholder}'`,
```

We should probably be able to the if condition, sa template string should do the checks & toString() call for us.